### DISCUSSION TITLE
multiple shortcuts for the same command

### DISCUSSION THREAD

**Discussion by csengh** [sumatrapdfreader/sumatrapdf#3592] [Q&A] [answered]
hy!
In manual's say:

> without modifiers, case do matter i.e. a and A are different

But I want to assign both case of the letter for a command, how can i achieve it?

I tried using:
-  a|A 
- a OR A

 but unfortunately does not work.

**Comment by GitHubRulesOK** [comment, marked as answer]
try "AND" = both
```
Shortcuts [
	[
		Cmd = foobar
		Key = a
	]
	[
		Cmd = foobar
		Key = A
	]
]
```

**Comment by csengh** [comment]
thanks, it works
